### DISCUSSION TITLE
short genome sequences can't load into JBrowse2

### DISCUSSION THREAD

**Discussion by Hanmingzi-Liu** [GMOD/jbrowse-components#2214] [General]
I loaded 23 genomes  into JBrowse2, but the final result can't show all of the genome sequences. It seems like that the short sequences can't show.  I want to know if somebody else have the same problem with me and how to solve this problem. Thanks for your help
![fig](https://user-images.githubusercontent.com/56110846/129181889-88958de9-1ea5-4287-aa3f-293643fcf738.png)
！

**Comment by garrettjstevens** [comment]
I'm not sure what exactly the problem you're describing is. Can you describe what you expect to see compared to what you actually see? Alternatively, what about the screenshot you posted is unexpected?

**Reply by Hanmingzi-Liu** [reply]
Thank you for your reply. Ok, I know what the problem is. I made a mistake. I mean that, for example, this genome is actually comprised of 1398 sequences, but the page only shows a total of 100 sequences. Now I know it is because that the drop-down list only show 100 entries! I can still search the result of the other entries by typing. So are there any way to list all of the sequences so users can select one conveniently?
![image](https://user-images.githubusercontent.com/56110846/129661856-0a84084b-2d79-469e-b058-2701d34c5bc3.png)
![image](https://user-images.githubusercontent.com/56110846/129662219-b0e974dc-b585-465a-bde3-df0aea1a2e20.png)

**Reply by garrettjstevens** [reply]
We don't show all sequences because there might be so many that it makes the browser page slower. Currently that limit is not configurable, either.
